mysite/stashEditor/templates/stashEditor/create.html

```diff
@@ -43,17 +43,20 @@ <h3>Idea editor</h3>
 <div class="row">
     <div class="col-sm-5">
         {% load crispy_forms_tags %}
-        <form action="" method="post" id="idea_form">
+        <form action="" method="post" id="idea_form" enctype="multipart/form-data">
             {% csrf_token %}
             {{ideaform.idea_title|as_crispy_field}}
             {{ideaform.idea_description|as_crispy_field}}
             {{categoryform.category_name|as_crispy_field}}
             <strong class="text-info" id="info" name="category_list"></strong>
             {{ideaform.idea_category|as_crispy_field}}
+            <strong class="text-info" id="info" name="image_upload">Choose image</strong>
+            <input type="file" name="image" value="test">
+            <button type="submit" name="button">Upload image</button>
             {{ideaform.is_public|as_crispy_field}}
         </form>
-        <button class="btn btn-warning" id="submit_idea_form">Submit</button>
```

Comment:
Button måste vara utanför form med en jquery function .onclick för submit. När man lägger till en ny kategori så skriver man namn på kategori och sen enter för att lägga till och då kan man skriva in en ny kategori. Om button ligger i form så blir det submit på enter

Reply:
Done